src/myapi.js

```diff
@@ -0,0 +1,52 @@
+const gameUrl = 'https://us-central1-js-capstone-backend.cloudfunctions.net/api/games/';
+const scoresUrl = 'https://us-central1-js-capstone-backend.cloudfunctions.net/api/games/oAHBEe3vcJhZ3vMizAyF/scores/';
+
+const newGame = async () => {
+  const newGameName = 'My new game';
+  const response = await fetch(gameUrl, {
+    method: 'POST',
+    body: JSON.stringify({ name: newGameName }),
+    headers: {
+      'Content-Type': 'application/json; charset=utf-8',
+    },
+  });
+  const jsonData = await response.json();
+  return jsonData;
+};
+
+const newUser = async (userValue, scoreValue) => {
```

Comment:
- [ ] This function name is not referring to its purpose, it is supposed to send a user's score to the server so it would be better to use a descriptive name. A function name should be a verb or a phrase fully exposing the intent behind it as well as the intent of the arguments. Let's fix that
- [ ] You can find more details about javascript clean code [here](https://devinduct.com/blogpost/22/javascript-clean-code-best-practices)

Reply:
The function name has been changed as requested. Thanks for the review